Makefile

```diff
@@ -81,9 +81,6 @@ all: format $(addsuffix -el.elf,$(TARGETS)) $(addsuffix -eb.elf,$(TARGETS)) gene
 	ln -srf testdata/loader-$(CLANG)-el.elf testdata/loader-el.elf
 	ln -srf testdata/loader-$(CLANG)-eb.elf testdata/loader-eb.elf
 
-# $BPF_CLANG is used in go:generate invocations.
-generate: export BPF_CLANG := $(CLANG)
```

Comment:
Doesn't this break `make generate` outside of container?

Reply:
Not for me, it defaults to `clang` and `llvm-strip`. I also sent a patch that removed a hardcoded clang-14 somewhere.